subsystem/operators_test.go

```diff
@@ -402,29 +402,6 @@ var _ = Describe("Operators endpoint tests", func() {
 
 			Expect(operatorSubscriptionName).To(Equal(lvm.LvmsSubscriptionName))
 		})
-
-		It("should lvm have right subscription name on 4.11", func() {
-			cluster := registerNewCluster(
-				"4.11.0",
```

Comment:
Removing otherwise valid tests is not the correct way to deal with errors.
The error was that a release image wasn't present for 4.11.0 it feels like the correct solution would be to add the wiremock stubs to ensure a 4.11 image is present and/or update this version if necessary.

Reply:
I removed this test as 4.11 is `EOL` now and this test tests specific behavior of lvm in 4.11

Reply:
I believe this test is no longer relevant

Reply:
Let's keep it until we also remove the [related code](https://github.com/openshift/assisted-service/blob/7cac29ddf85d31912a091d6a9c4bd453ee6a0003/internal/operators/lvm/manifest.go#L40-L57)

Reply:
Yes you are right, I will change the version

Reply:
I just set it to 4.11, then it will take the latest 4.11 release image